lib/drivers/weitai/weiTaiUSB.js

```diff
@@ -0,0 +1,1565 @@
+
+    const crc8_c = cRC8(
+      Buffer.concat(
+        [commandBody.slice(0, 3), commandBody.slice(4)],
+        commandBody.length - 1
+      )
+    );
+
+    if (crc8 != crc8_c) {
+      //CRC8校验不通过
+      return cb({ code: 'E_READ_FILE' }, null);
+    }
+
+    if (packet.length < command.length + length) {
+      //长度校验不通过
+      return cb({ code: 'E_READ_FILE' }, null);
+    }
+
+    const data = packet.slice(command.length, command.length + length);
+
+    const payload = data.slice(0, data.length - 16);
+    const md5 = data.slice(data.length - 16, data.length);
+
+    const md5_c = crypto
+      .createHash('md5')
+      .update(payload)
+      .digest();
+
+    if (!md5.equals(md5_c)) {
+      //md5校验不通过
+      return cb({ code: 'E_READ_FILE' }, null);
+    }
+
+    let inComeRes = WeitaiUSB.parsePayload(payload, cfg, cb);
+
+    return inComeRes;
+  }
+
+  static parsePayload(payload, cfg, cb) {
+    //每22字节为一条数据
+
+    let inComeRes = {
+      BloodGlucoses: [],
+      BasalRates: [],
+      BolusRates: [],
+      lastBasals: [],
+      alarm: [],
+    };
+
+    if (payload.length == 0) {
+      // 无数据
+      return inComeRes;
+      // return cb({code:'E_READ_FILE'}, null);
+    }
+
+    if (payload.length % 28) {
+      //数据异常，非22的倍数
+      return cb({ code: 'E_READ_FILE' }, null);
+    }
+
+    for (let i = 0; i < payload.length; i += 28) {
+      const history = payload.slice(i, i + 28);
+
+      const ID = history.slice(0, 4);
+      const SN = history.slice(4, 10);
+      const dateTime = history.slice(10, 16);
+      const status = history.slice(16, 22);
+      const event = history.slice(22, 28);
+
+      const recordID = uintFromArrayBuffer(ID, true);
+
+      const year = uintFromArrayBuffer(dateTime.slice(0, 1), true) + 2000;
+      const month = uintFromArrayBuffer(dateTime.slice(1, 2), true);
+      const day = uintFromArrayBuffer(dateTime.slice(2, 3), true);
+      const hour = uintFromArrayBuffer(dateTime.slice(3, 4), true);
+      const minute = uintFromArrayBuffer(dateTime.slice(4, 5), true);
+      const second = uintFromArrayBuffer(dateTime.slice(5, 6), true);
+
+      const battery = uintFromArrayBuffer(status.slice(0, 1), true);
+      const reservoir = uintFromArrayBuffer(status.slice(1, 2), true);
+      const basalRate = uintFromArrayBuffer(status.slice(2, 4), true);
+      const bolusRate = uintFromArrayBuffer(status.slice(4, 6), true);
+
+      const eventIndex = uintFromArrayBuffer(event.slice(0, 2), true);
+      const eventPort = uintFromArrayBuffer(event.slice(2, 3), true);
+      const eventType = uintFromArrayBuffer(event.slice(3, 4), true);
+      const eventUrgency = uintFromArrayBuffer(event.slice(4, 5), true);
+      const eventValue = uintFromArrayBuffer(event.slice(5, 6), true);
+
+      const timeText =
+        year +
+        '-' +
+        (month + 100).toString().substring(1) +
+        '-' +
+        (day + 100).toString().substring(1) +
+        ' ' +
```

Comment:
```suggestion
        'T' +
```
A 'T' is needed here to correct the date/time format.